R/count.R

```diff
@@ -23,17 +31,39 @@
 #'
 #' test_df %>%
 #'   count.(where(is.character))
-count. <- function(.df, ...) {
+#' 
+#' test_df %>%
+#'   count.(z, wt = y, name = "Sum of Y")
+#' 
+#' test_df %>%
+#'   count.(z, sort = TRUE)
+count. <- function(.df, ..., wt = NULL, sort = FALSE, name = NULL) {
   UseMethod("count.")
 }
 
 #' @export
-count..data.frame <- function(.df, ...) {
+count..data.frame <- function(.df, ..., wt = NULL, sort = FALSE, name = NULL) {
 
   .df <- as_tidytable(.df)
 
   .by <- enquos(...)
+  wt <- enquo(wt)
+  
+  if(quo_is_null(wt)){
+    .df <- summarize.(.df, N = .N, .by = c(!!!.by))
+  } else {
+    .df <- summarize.(.df, N = sum(!!wt), .by = c(!!!.by))
+  }
```

Comment:
Can you put
```r
if (is.null(name)) {
  name <- "N"
}
```
above the `summarize.()` calls, and then change the `summarize.()` calls to access the name like this:
```r
summarize.(.df, !!name := .N, .by = c(!!!.by))`
```